CHANGES_UNRELEASED.md

```diff
@@ -18,3 +18,11 @@ Use the template below to make assigning a version number during the release cut
   - Description of the change with a link to the pull request ([#0000](https://github.com/mozilla/application-services/pull/0000))
 
 -->
+
+## Tabs
+
+### ⚠️ Breaking changes ⚠️
+
+Tabs has been Uniffi-ed.
+
+`setLocalTabs` is now `updateLocalState`.
```

Comment:
As we discussed, we will just rename the function locally to match what android expects.